www/install/step_upgrade/process/process_step5.php

```diff
@@ -37,15 +37,18 @@
 require_once __DIR__ . '/../../../../bootstrap.php';
 
 $parameters = filter_input_array(INPUT_POST);
-if ((int)$parameters["send_statistics"] == 1) {
-    $query = "INSERT INTO options (`key`, `value`) VALUES ('send_statistics', '1')";
-} else {
-    $query = "INSERT INTO options (`key`, `value`) VALUES ('send_statistics', '0')";
-}
 
-$db = $dependencyInjector['configuration_db'];
-$db->query("DELETE FROM options WHERE `key` = 'send_statistics'");
-$db->query($query);
+if ($parameters) {
```

Comment:
merge those 2 lines in a single line : if (isset($parameters["send_statistics"]) && $parameters["send_statistics"] == '1') {

Reply:
No because in all case I don't want to execute both sub condition.
May be I can replace by:

if (isset($parameters["send_statistics"]) && (int)$parameters["send_statistics"] == 1) {
...
} elseif (isset($parameters["send_statistics"]) {
...
}